code/modules/jobs/job_types/bos.dm

```diff
@@ -279,6 +279,7 @@ Head Scribe
 		/obj/item/clothing/mask/gas/adv/gasmask = 1,
 		/obj/item/stock_parts/cell/ammo/ec = 2,
 		/obj/item/gun/energy/laser/plasma/glock = 1,
+		/obj/item/storage/box/bos = 1
```

Comment:
if you plan on giving the box to head scribe you might aswell give them the recipes that 1 - 4 get, but thats majority up to you. 

	for(var/datum/crafting_recipe/recipe as() in GLOB.knight_gunsmith_recipes)
		H.mind.teach_crafting_recipe(recipe)